{
  "active": false,
  "connections": {
    "Error Trigger": {
      "main": [
        [
          {
            "node": "Telegram",
            "type": "main",
            "index": 0
          }
        ]
      ]
    }
  },
  "createdAt": "2023-04-20T08:34:22.191Z",
  "id": "3",
  "name": "Global Error Monitor",
  "nodes": [
    {
      "parameters": {},
      "id": "8d1dd34a-67d4-4e92-a6dd-503675e1b3f2",
      "name": "Error Trigger",
      "type": "n8n-nodes-base.errorTrigger",
      "typeVersion": 1,
      "position": [
        640,
        420
      ]
    },
    {
      "parameters": {
        "chatId": "=-1001843718451",
        "text": "=#Error\n\nWorkFlow: {{ $json[\"workflow\"][\"name\"] }}\nError: {{ $json[\"execution\"][\"error\"][\"message\"] }}\nUrl: {{ $json[\"execution\"][\"url\"] }}\nStacktrace: {{ $json[\"execution\"][\"error\"][\"stack\"] }}",
        "additionalFields": {}
      },
      "id": "251c1754-9441-4fed-bdae-18303584b71e",
      "name": "Telegram",
      "type": "n8n-nodes-base.telegram",
      "typeVersion": 1,
      "position": [
        860,
        420
      ],
      "credentials": {
        "telegramApi": {
          "id": "1",
          "name": "Telegram account"
        }
      }
    }
  ],
  "pinData": {},
  "settings": {},
  "staticData": null,
  "tags": [],
  "triggerCount": 0,
  "updatedAt": "2023-04-23T01:20:09.000Z",
  "versionId": "a16d2ae4-0e80-4b21-b18d-6dc20175a7ae"
}